### ISSUE TITLE
[BUG] only able to pass 1 param when calling raise_event.

### ISSUE THREAD

**Issue by RobWMoore** [OpenSIPS/opensips#2669]
**OpenSIPS version you are running**

```
version: opensips 3.3.0-dev (x86_64/linux)
flags: STATS: On, DISABLE_NAGLE, USE_MCAST, SHM_MMAP, PKG_MALLOC, Q_MALLOC, F_MALLOC, HP_MALLOC, DBG_MALLOC, CC_O0, FAST_LOCK-ADAPTIVE_WAIT
ADAPTIVE_WAIT_LOOPS=1024, MAX_RECV_BUFFER_SIZE 262144, MAX_LISTEN 16, MAX_URI_SIZE 1024, BUF_SIZE 65535
poll method support: poll, epoll, sigio_rt, select.
git revision: 481ac49
main.c compiled on  with gcc 10
```

**Describe the bug**

When calling "raise_event" core function (either via opensips-cli or http MI json rpc) I only seem to be able to pass one param.

**To Reproduce**

add the following event route to the cfg script (don't forget to register it somewhere!) 

```
event_route[E_TEST] {
	xlog("L_ERR", "Test Event raised \r\n");
	xlog("first parameters is $param(1)\r\n");
	xlog("Second parameters is $param(2)\r\n");
        xlog("IP is $param(ip)\r\n");
}
```
attempt to call the Event route from either opensips-cli or http mi

opensips-cli command:
```
opensips-cli -x mi raise_event E_TEST '{"id":"test", "ip":"192.168.0.1"}'
```
jsonrpc body for http mi post:
```
{"jsonrpc": "2.0", "id": "18293", "method": "raise_event", "params": ["E_TEST",["cheese","cheese2"]]}
```
or:
```
{"jsonrpc": "2.0", "id": "18293", "method": "raise_event", "params": ["E_TEST", {"id":"cheese", "ip": "192.168.0.1"}]}
```

logs will looks something like this (dependant on mi method above used):
```
Nov  1 14:31:33 [82] Test Event raised
Nov  1 14:31:33 [82] first parameters is {id:test, ip:192.168.0.1}
Nov  1 14:31:33 [82] WARNING:event_route:event_route_param_get: Parameter 2 not found for event E_TEST - max 2
Nov  1 14:31:33 [82] Second parameters is <null>
Nov  1 14:31:33 [82] WARNING:event_route:event_route_param_get: Parameter <ip> not found for event <E_TEST>
```
**Expected behavior**

All params passed to be accessible in route via either $param(1) or name reference $param(ip)

**Comment by razvancrainea** [comment]
@RobWMoore thanks for the report - there are actually two issues here:

1. there was a bug duplicating the params passed by the `raise_event` function, resulting in always passing just one parameter to the route - this has been fixed in 3a9b4a1
2. the second problem is related to `opensips-cli` - the current code presumes that the parameters are always positional - therefore, they will never be treated as JSON, but as strings. This is what [this line](https://github.com/OpenSIPS/opensips-cli/blob/master/opensipscli/modules/mi.py#L46) does - deleting it and re-installing the cli will allow you send a json as parameter. The problem however is that if we delete the line, you won't be able to pass parameters as positional arguments.

To solve this, one solution that I am thinking of is to change the interface of `event_raise` MI function to specify the json parameters in a separate node (rather than `params`), such as `json_params`. This way we can know exactly how the param should be built. This is just a thought, any feedback regarding this is welcome.

**Comment by razvancrainea** [comment]
I've just pushed a new commit in OpenSIPS CLI (OpenSIPS/opensips-cli@696114a) that allows you to use the `-j` modifier to specify the parameters as JSON, rather than converting them to an array. See the [documentation](https://github.com/OpenSIPS/opensips-cli/blob/master/docs/modules/mi.md#modifiers) for more details and [examples](https://github.com/OpenSIPS/opensips-cli/blob/master/docs/modules/mi.md#examples) for a few ways you can use.
